portal-api/src/v1/roles/roles.js

```diff
@@ -0,0 +1,12 @@
+const ROLES = {
+  MAKER: 'maker',
+  CHECKER: 'checker',
+  ADMIN: 'admin',
+  UKEF_OPERATIONS: 'ukef_operations',
+  READ_ONLY: 'read-only',
+  EDITOR: 'editor',
+  DATA_ADMIN: 'data-admin',
+  INTERFACE: 'interface',
```

Comment:
Are we missing EFM here by the way? It's present in portal ui

Reply:
It's not used in portal-api